.storybook/recipes/Sliders.stories.tsx

```diff
@@ -195,7 +195,7 @@ export const WithMultipleVisualLabels: StoryObj<Args> = {
             id="slider-label"
             text="Mood Slider"
           />
-          <div className="flex w-1/2 justify-between py-4 text-center text-h1">
+          <div className="text-h1 flex w-1/2 justify-between py-4 text-center">
```

Comment:
This is just reordering the class names so it doesn't throw lint errors... @booc0mtaco

Reply:
Hm, I had to do this on several PRs too. I think that minor version of tailwind contained this breaking change :(